tests/integration/support/test_support.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+
+from tests.integration.helpers import assert_headers_in_lines, exec_test_command
+
+BASE_CMD = ["linode-cli", "tickets"]
+
+
+# this will create a support ticket on your account
+def test_create_support_ticket(create_linode):
+    linode_id = create_linode
+    exec_test_command(
+        BASE_CMD
+        + [
+            "create",
+            "--description",
+            "Creating support ticket for test verification",
+            "--linode_id",
+            linode_id,
+            "--summary",
+            "Testing ticket" "--text",
+            "--no-headers",
+        ]
+    )
+
+
+def test_tickets_list():
+    res = (
+        exec_test_command(BASE_CMD + ["list", "--text", "--delimiter=,"])
+        .stdout.decode()
+        .rstrip()
+    )
+    lines = res.splitlines()
+    headers = ["summary", "opened_by", "opened"]
+    assert_headers_in_lines(headers, lines)
+
+
+@pytest.fixture
+def get_ticket_id():
```

Comment:
Small thing, it is best practice to name the fixtures with descriptive names or prefix the fixtures instead of starting from a verb. So in this case, I think the something like `ticket_id` would be appropriate.